README.md

```diff
@@ -0,0 +1,27 @@
+# AI-IaC-Prompts
```

Comment:
Are you planning to move this repo under the "Azure" github org before sharing with customers?

Reply:
Yes, that would be the plan.